File: exercicios/R/1175/1175.r
input = file('stdin','r')

vector <- c(20)

for(i in 20:1){
    num = as.integer(readLines(input,n=1))

    vector[i] = num
}

for(j in 1:20){
    write(sprintf("N[%d] = %d",j-1,vector[j]),'')
}

File: exercicios/JavaScript/1175/1175.js
var input = require('fs').readFileSync('stdin','utf8');
var lines = input.split('\n');

var list = [20];

for(let i=19; i>=0; i--){
    var num = parseInt(lines.shift());

    list[i] = num;
}
for(let j=0; j<20; j++){
    console.log("N[%d] = %d", j,list[j]);
}